accounts-db/src/accounts.rs

```diff
@@ -44,8 +44,25 @@ use {
     },
 };
 
+// max number of accts an entry can hold: 64 * 256
+const ENTRY_ACCTS_LOOKUP_TABLE_SIZE: usize = 16384;
 pub type PubkeyAccountSlot = (Pubkey, AccountSharedData, Slot);
 
+#[derive(Debug, Default)]
+struct EntryAcctLocks {
```

Comment:
nit: i prefer `EntryAccountLocks`, considering this is a close kin to already existing `AccountLocks`.